### ISSUE TITLE
Atom fell when open pdf

### ISSUE THREAD

**Issue by skilllet** [izuzak/atom-pdf-view#142]
Men my atom has felled. when I try open some pdf file. If you want to see my config or others thing i can send you it. ( Sorry about my English )

**Comment by izuzak** [comment]
> Men my atom has felled. when I try open some pdf file. If you want to see my config or others thing i can send you it.

Thanks for opening an issue, but this doesn't give any information about the problem you observed, which versions of Atom and this package you're using, the OS and version you're using, how to reproduce the problem nor the PDF which caused the issue. If you can provide detailed information about that so that we can try reproducing the issue and perhaps fixing -- I'd be happy to reopen this.

**Comment by skilllet** [comment]
Yes men you right, I send all details, if you want somthing written my.

Now I can say that         os:  win7 64-bit SP1 ( Flaty )
                                    atom version:  1.8.0
                                    plagin-version:   0.50.0
I attach the problem file

2016-07-13 12:08 GMT+03:00 Ivan Žužak <notifications@github.com>:

**Comment by izuzak** [comment]
There's no attachment here -- can you attach it with a new comment on this issue, instead of replying via email?

See these instructions:

https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/

**Comment by izuzak** [comment]
@skilllet I just tried opening the PDF you provided in https://github.com/izuzak/atom-pdf-view/issues/143 and I can see it just fine and nothing crashed:

<img width="1129" alt="screen shot 2016-07-17 at 17 54 09" src="https://cloud.githubusercontent.com/assets/38924/16903873/7ff4559c-4c47-11e6-9a08-24cdd9b047aa.png">

Can you share a screenshot which demonstrates the problem you're having with that file?
